### PR TITLE
[15.0] sale_commission_formula: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**

### CODE REVIEW

**Comment by HaraldPanten** [comment]
Hi @AungKoKoLin1997 will you continue with the migration?

THX

**Comment by AungKoKoLin1997** [comment]
> Hi @AungKoKoLin1997 will you continue with the migration?

Yes. The migration of commission modules are done. So, I am planning to work on these days.

**Comment by HaraldPanten** [comment]
> > Hi @AungKoKoLin1997 will you continue with the migration?
> 
> Yes. The migration of commission modules are done. So, I am planning to work on these days.

THX!

**Comment by AungKoKoLin1997** [comment]
@HaraldPanten Can you please review this PR?

**Comment by HaraldPanten** [comment]
> @HaraldPanten Can you please review this PR?

Sure!

@luis-ron

**Review by luis-ron — APPROVED** [review]
LGTM

File: commission_formula/views/commission_view.xml [hunk 1]
```diff
@@ -29,13 +17,13 @@
                     <div attrs="{'invisible':[('commission_type','!=','formula')]}">
                         <h2 name="hr_instructions">Instructions</h2>
                         <span class='instructions'>
-  Use 'line' to access all the line fields.
-  As field names may vary, you should differentiate the
-  formulas for sale.order.line and account.move.line.
-  You can, for instance, use line._name == 'sale.order.line'
-  or line._name == 'account.move.line'.
+                            Use 'line' to access all the line fields.
```
Comment (pedrobaeza) [inline commission_formula/views/commission_view.xml]: Why this indentation?

Comment (AungKoKoLin1997) [reply commission_formula/views/commission_view.xml]: I did it for more readable code. But I didn't expect that will effect the UI. I reverted the changes.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**